test/pdo_sqlsrv/pdo_commit.phpt

```diff
@@ -11,21 +11,29 @@ starts a transaction, insert 2 rows and commit the transaction
     $conn->exec("IF OBJECT_ID('Table1', 'U') IS NOT NULL DROP TABLE Table1");
     $conn->exec("CREATE TABLE Table1(col1 CHARACTER(1), col2 CHARACTER(1)) ");
    
-    $conn->beginTransaction();
     $ret = $conn->exec("insert into Table1(col1, col2) values('a', 'b') ");
     $ret = $conn->exec("insert into Table1(col1, col2) values('a', 'c') ");
    
     //revert the inserts
-    $ret = $conn->exec("delete from Table1 where col1 = 'a'");
+    $conn->beginTransaction();
+    $rows = $conn->exec("delete from Table1 where col1 = 'a'");
     $conn->commit();
-    echo $ret." rows affected\n";
+    echo $rows." rows affected\n";
+    
+    $stmt = $conn->query("select * from Table1");
+    if ( count( $stmt->fetchAll() ) == 0 )
+        echo "Transaction committed successfully\n";
+    else
+        echo "Transaction failed to commit\n";
    
     //drop the created temp table
     $conn->exec("DROP TABLE Table1 ");
    
     //free statement and connection
     $ret=NULL;
+    $stmt = NULL;
     $conn=NULL;
```

Comment:
the spacing of $stmt = null is inconsistent with others
also, PDO::exec() returns integer, so you don't need to set it to null ;)